assets/css/sensei-course-theme/colors.scss

```diff
@@ -0,0 +1,12 @@
+$sensei-module-lesson: #101517;
```

Comment:
I think we should use CSS variables here and centralize all of them on the lines below.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/6783/commits/d481901ac06c00a7bdb031dc23030306fac2879b.